src/layer/marker/Marker.js

```diff
@@ -235,7 +235,7 @@ export var Marker = Layer.extend({
 
 		if (options.keyboard) {
 			icon.tabIndex = '0';
-			icon.setAttribute('role', 'button');
+			icon.setAttribute('role', 'img');

```

Comment:
For `<img>` markers this `role` is redundant (because that element is implicitly `role="img"`).

But I'm not sure it's desired for divIcons either, because they can have custom HTML, so implementers may end up with invalid HTML (the same consideration applies to the use of `role="button"` and `aria-expanded`  for divIcons). Perhaps it should be their responsibility to not use elements that are disallowed inside `img` (all) or buttons (interactive elements) per the HTML spec, but that'd restrict the kind of HTML elements that should be used in the custom HTML in divIcons.

@mourner, @IvanSanchez, @Falke-Design, I believe we need to decide whether divIcons should be as accessible as regular markers by default but which may result in invalid HTML if the implementer isn't careful, or simply not have the same accessible defaults.